packages/analytics/analytics-geo-map/src/index.ts

```diff
@@ -0,0 +1,17 @@
+// import type { App } from 'vue'
+import AnalyticsGeoMap from './components/AnalyticsGeoMap.vue'
+
+// Export Vue plugin
```

Comment:
nit: you can delete all this plugin init code

Reply:
done